dockers/docker-orchagent/supervisord.conf.j2

```diff
@@ -19,6 +19,13 @@ autostart=true
 autorestart=unexpected
 buffer_size=1024
 
+[eventlistener:supervisor-proc-watchdog-listener]
+command=/usr/bin/supervisor-proc-watchdog-listener --container-name swss
```

Comment:
>listener [](http://example.com/codeflow?start=42&length=8)

Could you explore how much code could be reused if combined this listener with above "supervisor-proc-exit-listener"?

Reply:
Fixed, merged code to supervisor-proc-exit-listener